Tests/DbgEngTest/TestBase.cs

```diff
@@ -46,11 +45,42 @@ protected static void Initialize(string dumpFile, string symbolPath, bool addSym
             {
                 symbolPath += ";srv*";
             }
+        }
+
+        /// <summary>
+        /// Initializes the test class with the specified dump file.
+        /// </summary>
+        /// <param name="dumpFile">The dump file.</param>
+        /// <param name="symbolPath">The symbol path.</param>
+        /// <param name="addSymbolServer">if set to <c>true</c> symbol server will be added to the symbol path.</param>
+        protected static void InitializeDump(string dumpFile, string symbolPath, bool addSymbolServer = true)
+        {
+            NormalizeDebugPaths(ref dumpFile, ref symbolPath, addSymbolServer);
 
             client = DebugClient.OpenDumpFile(dumpFile, symbolPath);
             Context.Initalize(client);
         }
 
+        /// <summary>
+        /// Initializes the test class with the specified process file.
+        /// </summary>
+        /// <param name="processPath"></param>
+        /// <param name="symbolPath"></param>
+        /// <param name="addSymbolServer"></param>
+        /// <param name="debugEngineOptions"></param>
```

Comment:
Done.

---
In reply to: [69627676](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r69627676) [](ancestors = 69627676)